Examples/Detectors/TGeoDetector/include/ActsExamples/TGeoDetector/TGeoDetector.hpp

```diff
@@ -102,6 +104,10 @@ struct TGeoDetector : public ActsExamples::IBaseDetector {
       unsigned int cylinderNPhiSegments;
       unsigned int discNRSegments;
       unsigned int discNPhiSegments;
+
+      bool itkModuleSplit = false;
```

Comment:
can we rename this `applyModuleSplit` in order not to have `ITk` specifically mentioned here?

Reply:
That's a good point. Right now it is ITk specific, because we map the splitting parameters by sensor name. However, this could be separated from the actual splitting functions. These could be put into a new splitter or maybe merged with the cylinderDiscSplitter? Then the ITk config part could e.g. be moved to a wrapper around that splitter. What do you think?